packages/esm-bed-management-app/src/bed-administration/form/add-edit-bed.workspace.tsx

```diff
@@ -0,0 +1,394 @@
+                onChange={field.onChange}
+                invalid={!!errors.occupancyStatus?.message}
+                invalidText={errors.occupancyStatus?.message}>
+                <SelectItem text={t('selectOccupancyStatus', 'Select occupancy status')} value="" />
+                {occupancyStatuses.map((status, index) => (
+                  <SelectItem key={`occupancy-${index}`} text={t(status.toLowerCase(), status)} value={status} />
+                ))}
+              </Select>
+            )}
+          />
+        </ResponsiveWrapper>
+
+        <ResponsiveWrapper>
+          <Controller
+            control={control}
+            name="bedType"
+            render={({ field }) => (
+              <Select
+                id="bedType"
+                labelText={t('bedType', 'Bed Type')}
+                value={field.value}
+                onChange={field.onChange}
+                invalid={!!errors.bedType?.message}
+                invalidText={errors.bedType?.message}>
+                <SelectItem text={t('selectBedType', 'Select bed type')} value="" />
+                {availableBedTypes.map((bedType, index) => (
+                  <SelectItem key={`bedType-${index}`} text={bedType.name} value={bedType.name} />
+                ))}
+              </Select>
+            )}
+          />
+        </ResponsiveWrapper>
+
+        <ResponsiveWrapper>
+          <Controller
+            control={control}
+            name="bedTags"
+            render={({ field: { onChange, onBlur, value, ref } }) => (
+              <div>
+                <ComboBox
+                  id="bedTags"
+                  titleText={t('bedTags', 'Bed Tags')}
+                  placeholder={t('selectBedTags', 'Select bed tags')}
+                  items={availableBedTags}
+                  itemToString={(item) => item?.name ?? ''}
+                  selectedItem={null}
+                  onChange={({ selectedItem }) => {
+                    if (
+                      selectedItem &&
+                      !value?.some((tag) => (tag.id || tag.uuid) === (selectedItem.id || selectedItem.uuid))
+                    ) {
+                      onChange([...(value || []), selectedItem]);
+                    }
+                  }}
+                  onBlur={onBlur}
+                  ref={ref}
+                  invalid={!!errors.bedTags?.message}
+                  invalidText={errors.bedTags?.message}
+                />
+              </div>
+            )}
+          />
+        </ResponsiveWrapper>
+      </div>
+
+      <ButtonSet
+        className={classNames({
+          [styles.tablet]: isTablet,
+          [styles.desktop]: !isTablet,
+        })}>
+        <Button className={styles.buttonContainer} kind="secondary" onClick={() => closeWorkspace()}>
+          {t('cancel', 'Cancel')}
+        </Button>
+        <Button
+          disabled={isSubmitting || !isDirty || !hasLocations}
+          className={styles.buttonContainer}
```

Comment:
```suggestion
          className={styles.button}
```

Isn't `.button` a better name here?